packages/cli/scripts/build.mjs

```diff
@@ -30,10 +31,25 @@ await compileDevTemplates();
 const pkgPath = join(process.cwd(), 'package.json');
 const pkg = JSON.parse(readFileSync(pkgPath, 'utf8'));
 const externals = Object.keys(pkg.dependencies || {});
-await esbuild({
-  bundle: true,
-  external: externals,
-});
+await Promise.all([
```

Comment:
Is this change still necessary? I thought we decided on implementing as a CLI subcommand?